README.md

```diff
@@ -245,6 +246,29 @@ You can populate below template according to your requirements and use it as you
 
 # WEBDAV_URL_INSECURE="true"
 
+# You can also backup files to any SSH server:
+
+# The URL of the remote SSH server
+
+# SSH_HOST_NAME="server.local"
```

Comment:
Hostname usually is spelled as a single word, so I'd probably make this `SSH_HOSTNAME`, also `SSHHostname`

Reply:
I used naming as in the `.ssh/config` file.

```
Host github.com
        HostName github.com
        User git
        IdentityFile ~/.ssh/github.com/id_rsa
```

Reply:
Good point, I never noticed it's spelled like this in there. Then let's keep it `HOST_NAME` I guess.